json-endpoints/class.wpcom-json-api-get-site-endpoint.php

```diff
@@ -289,6 +289,9 @@ public function build_current_site_response( ) {
 
 				if ( $is_jetpack ) {
```

Comment:
@lezama or @enejb 
why is `$is_jetpack` ignored?
everything in the conditional is not getting returned in the endpoint. :(